google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1433,10 +1433,15 @@ def compose sources, destination, acl: nil, encryption_key: nil
         #   use the signed URL.
         # @param [String] issuer Service Account's Client Email.
         # @param [String] client_email Service Account's Client Email.
-        # @param [OpenSSL::PKey::RSA, String] signing_key Service Account's
-        #   Private Key.
-        # @param [OpenSSL::PKey::RSA, String] private_key Service Account's
-        #   Private Key.
+        # @param [OpenSSL::PKey::RSA, String, Proc] signing_key Service Account's
+        #   Private Key or a Proc that accepts a single parameter which is used
```

Comment:
Can you insert a description of the Proc's parameter? Maybe just something as simple as: `that accepts a single String parameter, the data to be signed, which is used...`

Reply:
I think it would also be good to clearly specify the Proc's return value. Maybe:

`a Proc that accepts a single String parameter, the data to be signed, and returns a valid RSA SHA256 signature String.`

Reply:
Great!